app/server/routes/api.js

```diff
@@ -373,4 +392,46 @@ module.exports = function(router) {
     SettingsController.updateWhitelistedEmails(emails, defaultResponse(req, res));
   });
 
+  /**
+   * [ADMIN/OWNER]
+   * Edits the hack's start time
+   */
+  router.put('/settings/checkInStart', isAdmin, function(req, res) {
+    var checkInOpen = req.body.checkInStart;
+    SettingsController.updateField('checkInOpen',checkInOpen, defaultResponse(req, res));
+  });
+
+  /**
+   * [ADMIN/OWNER]
+   * Edits the hack's allowed team size
+   */
+  router.put('/settings/teamSizeAccepted', isAdmin, function(req, res) {
+    var teamSize = req.body.teamSizeAccepted;
```

Comment:
Nitpicking: `teamSize` should be `teamSizeAccepted` to keep the naming scheme and prevent future errors due tu code clarity.